pkg/build/api/types.go

```diff
@@ -356,6 +356,12 @@ type ImageChangeTrigger struct {
 	// LastTriggeredImageID is used internally by the ImageChangeController to save last
 	// used image ID for build
 	LastTriggeredImageID string
+
+	// From is a reference to an ImageStreamTag that will trigger a build when updated
+	// It is optional. If no From is specified, the From image from the build strategy
+	// will be used. Only one ImageChangeTrigger with an empty From reference is allowed in
+	// a build configuration.
+	From *kapi.ObjectReference
```

Comment:
Future reference, please make sure you get API review prior to merging API changes - we're going to try to continue to do what upstream does and make sure there is a final review on all public API changes.

This is fine